### DISCUSSION TITLE
NOISE FOR 2,3 SECONDS

### DISCUSSION THREAD

**Discussion by aqildad-create** [pschatzmann/arduino-audio-tools#838] [Q&A] [answered]
I am using sbc decoder and encoder via espnow its is working but after random time I get loud wrestle for 2,3 seconds  and every thing get normal then No esp reset just loud wrestle .  

this loud wrestle can come at any time .

**Comment by pschatzmann** [comment, marked as answer]
Are you getting this with my examples ? 

My gut feeling is that you might have selected a sample rate which is too high and the data playback is running into some buffer underflows. There are also some parameters in the ESPNowStreamConfig that you can play with

**Comment by aqildad-create** [comment]
Yes , it is your example of SBC communication with some changes yo make it
for i2s input .

The sampling rate I select is low I thing  16000 and bit rate 16  (also I
try with 32000 and 44100 but with this 2 sampling I am getting noise only
and  ).

At 16000 sampling its working and sound quality is normal but at any time
getting noise for 2,3 seconds .

The noise some time time generate after 15,20 minutes or some time come
even within minute.

Sampling is 16000 so may be its not issue of high sampling .

And ok I will try to play other parameter also . .
